### PR TITLE
Added a course for Mongo DB by thapa technical

### PR DESCRIPTION
**Pull request by jarvis8**
## What does this PR do?
Add resource(s) | Remove resource(s) | Add info | Improve repo

## Checklist:
- [ ] Read our [contributing guidelines](https://github.com/EbookFoundation/free-programming-books/blob/main/docs/CONTRIBUTING.md).
- [ ] [Search](https://ebookfoundation.github.io/free-programming-books-search/) for duplicates.
- [ ] Include author(s) and platform where appropriate.
- [ ] Put lists in alphabetical order, correct spacing.
- [ ] Add needed indications (PDF, access notes, under construction).
- [ ] Used an informative name for this pull request.

## Follow-up

- Check the status of GitHub Actions and resolve any reported warnings!

### CODE REVIEW

**Comment by jarvis8** [comment]
please merge

File: courses/free-courses-hi.md [hunk 1]
```diff
@@ -311,6 +312,11 @@
 * [Statistics And Probability](https://www.youtube.com/playlist?list=PLU6SqdYcYsfLRq3tu-g_hvkHDcorrtcBK) - Gajendra Purohit
 
 
+### Mongo-Db
```
Comment (eshellman) [inline courses/free-courses-hi.md:315]: not "Mongo DB"???

**Comment by jarvis8** [comment]
please merge

**Comment by jarvis8** [comment]
please merge

**Pull request merged**